### DISCUSSION TITLE
用ai偵測憂鬱症前兆

### DISCUSSION THREAD

**Discussion by Jimmy911218** [1101-GS4719-C/Python-Community-GS4719-C#652] [科技新知]
在現在憂鬱症已經成為了常聽到的詞了，常常會在youtobe或其他地方聽到
如果能提早發覺自己有憂鬱傾向提早解決的話將可以造成一些良好的現象
這個城市有個優點就是他只記錄生應變化不會記錄內容寶慧人的隱私
但也是缺點聲音不一定能完美玉徹這個症狀
[[https://technews.tw/2022/06/07/depressed-this-algorithm-can-tell-from-the-tone-of-your-voice/](url)

**Comment by liangyinglly** [comment]
連結好像點不進去ㄟ

**Comment by Yuhsin000** [comment]
連結點不進去，不過我覺得這個技術很好，有些人看起來感覺狀況就不是太好的樣子，或許可以量化那些下垂的嘴角、緊湊的眉眼，提早發現他人的異常，拯救一條生命或靈魂

**Comment by Ben6uo** [comment]
鏈接點不進去 不過這個想法真的很強 可以拯救很多人

**Comment by minniewei** [comment]
我覺得現在的ai真是越來越萬能了，感覺所有事情都可以做到

**Comment by sys511613** [comment]
連結進不去，而且文章錯字好像有點多==
主辦單位可以管一下嗎

**Comment by leon10456** [comment]
錯字+沒用的連結...

**Comment by jason79461385** [comment]
憂鬱症現代人蠻多人都有的 希望這項技術能盡早開發完善 還有我的鏈接呢 老鐵

**Comment by Hchejxnc** [comment]
憂鬱症的確是現代人常遇到的問題，這項技術發展成熟後，或許就能及早發現及早治療了。

**Comment by harrylee1971** [comment]
ai在研究憂鬱症可能沒辦法完善，並且精神科的問題還是由人去判斷比較恰當，ai仍是一個沒靈魂的工具而已

**Comment by ghost** [comment]
連結點不進去喔，我記得我之前有看過一份實驗，就是用ai來判斷在此刻的人是否有自殺的念頭，好像結果很準，由此可知，人的情緒即便是經過隱藏，還是能透過微表情看出，而若憂鬱症患者在表情上有所不同，或許也能用ai來判斷也說部定

**Comment by Sylvie0601** [comment]
我覺得這真的是一項非常需要被發展的技術ㄟ，憂鬱症僅幾年才開始被重視，但憂鬱症若是不治療的話，惡化的情況很快，最後會導致患者自殺等等的憾事，我由衷的希望憂鬱症可以被及早治療，讓這些患者能夠早日脫離這個心理病痛。

**Comment by orianna901001** [comment]
我認同上面的回覆，憂鬱症確實越來越普遍，就連我自己上個通識課，都看到坐在自己附近的人在查相關資訊，希望她沒事QQ。

**Comment by Chienyuuu6** [comment]
跟大家分享五月是Mental Health awareness month!!!

我認為大部分的人認知到自己心裡可能出狀況的時候，是害怕向外尋求援助的，往往都會很隱密的差很多資料來確定自己是不是發生問題，如果能夠有這樣的機器讓我們自己先做初步判斷，並且針對症狀程度給予一些建議，或許需要幫助的人就得到那一股勇氣去面對自己的問題！

**Comment by a3240281370** [comment]
以現在AI的能力，透過AI技術分析和處理大量的數據，包括文字、語音和影像等，以尋找可能與憂鬱症相關的模式和特徵，確實是可發展的方向
但是憂鬱症是一個嚴重且需要敏感處理的問題，對於任何涉及憂鬱症的偵測工具，保護使用者的隱私和個人資訊至關重要。遵循嚴格的倫理和隱私準則是開發和應用這些技術的重要考慮因素。

**Comment by pigeon9975** [comment]
好好奇連結是甚麼喔喔喔，但都點不進去，作者可以help一波嗎？
現在社會運行得太快，資訊太豐富，常常一個不小心就失足，讓自己陷入憂鬱，然而，很多人不知道自己生病了，或是因為害怕社會觀感而不願去就醫，如果ai可以看出憂鬱症前兆，說不定能讓更多面對這種情況的患者獲得被拯救的機會！
再說一次，真的好好奇連結的內容是甚麼qq

**Comment by zephyrxin01** [comment]
我覺得憂鬱症患者通常不太有病識感。我親人之前跟我分享過他當時的狀況就是覺得自己只是抗壓性不夠，實驗熬過去就撐過來了。但中間因為疫情的關係，連關心身邊的人都變得有距離，所以他差一點就走不出來了。還好實驗室的另外一個同儕有發現他的狀況不對勁，回家休息了一陣子，情緒調適才變正常。

**Comment by alstinlin** [comment]
雖然用ai偵測憂鬱症前兆聽起來頗有難度，但我有看到同學的專題是利用ai影像辨識偵測病患需求，所以我覺得應該有機會運用到相同的概念，一方面也蠻好奇是怎麼實作的!
